regulations/templatetags/layers.py

```diff
@@ -0,0 +1,30 @@
+from django import template
+from django.urls import reverse
+from regulations.generator import api_reader
+
+register = template.Library()
+
+@register.filter(name='internalcitation')
+def internalcitation(value, arg):
+    if value == "":
+        return value
+
+    citation_template = template.loader.get_template('regulations/layers/internal_citation.html')
+    label = "-".join(arg['label'])
+    client = api_reader.ApiReader()
+    result = client.layer("internal-citations", "cfr", label, arg['version'])
+    if result is None:
+        return value
+    try:
+        citations = result[label]
+        citations.sort(key=lambda x: x["offsets"][0][0], reverse=True)
+        for citation in citations:
```

Comment:
change to `entry in citations`
```suggestion
        for entry in citations:
```